From 42c0dbceb05124babd1a5d3c7c42545960a4f9a6 Mon Sep 17 00:00:00 2001
From: Pierluigi Passaro <pierluigi.p@variscite.com>
Date: Fri, 15 Feb 2019 17:56:12 +0100
Subject: [PATCH 1/2] imx8-var-dart: compile imx-sdma driver as module

Starting from udev 217, kernel setting CONFIG_FW_LOADER_USER_HELPER
should be set to =n, because the userspace firmware loader has been
removed.
Yocto is avoiding the problem by reverting related patches in udev.
The easy way to avoid problems is compiling as modules all drivers
requiring firmware loaded from the filesystem.
This will help with Debian filesystems where udev is not patched.
---
 arch/arm64/configs/imx8_var_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/imx8_var_defconfig b/arch/arm64/configs/imx8_var_defconfig
index 45ff842b9a43..026dd319dc3a 100644
--- a/arch/arm64/configs/imx8_var_defconfig
+++ b/arch/arm64/configs/imx8_var_defconfig
@@ -440,7 +440,7 @@ CONFIG_RTC_DRV_SNVS=y
 CONFIG_RTC_DRV_IMX_SC=y
 CONFIG_DMADEVICES=y
 CONFIG_FSL_EDMA_V3=y
-CONFIG_IMX_SDMA=y
+CONFIG_IMX_SDMA=m
 CONFIG_MXS_DMA=y
 CONFIG_STAGING=y
 CONFIG_ION=y
-- 
2.17.1

